scaling_performance/routing_optimization.adoc

```diff
@@ -144,6 +145,15 @@ To modify the HAProxy configuration file, refer to
 xref:../install_config/router/customized_haproxy_router.adoc#install-config-router-customized-haproxy[Deploying
 a Customized HAProxy Router].
 
+==== Increasing the Number of Threads
+
+The HAProxy router comes with support for multithreading in {product-title}.
+On a multiple CPU core system, increasing the number of threads can help the performance,
+especially when terminating SSL on the router.
+
+To specify the number of threads for the HAProxy router,
+set the router environment variable `ROUTER_THREADS`.
```

Comment:
How about cross references to:

* https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/configuring_clusters/setting-up-a-router (3.2.25. Enable HAProxy Threading)
* https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/architecture/networking#env-variables?

This would be to avoid scattering duplicate documentation around.

Reply:
Thanks for looking, Dan. I was thingking about that, but don't like broken links. ;)  In any case, changed as you suggested.